lib/Sources/Tabs/TabGridModel.swift

```diff
@@ -92,6 +92,8 @@ public final class TabGridModel {
 
     public var isEmpty: Bool { tabs.isEmpty }
 
+    public func urlForIndex(index: Int) -> URL? { tabs[index]?.url }
+
```

Comment:
Continuing the above tweak, this would be better modelled as:
```suggestion
    public func findTab(for index: Int) -> TabModel? {
        tabs[index]
    }
```